### DISCUSSION TITLE
Web Scraping Consultancy

### DISCUSSION THREAD

**Discussion by JakeWalker23** [rl1987/trickster.dev#3] [General]
Hello @rl1987,

Came across Trickster.dev and really enjoyed your Web scraping posts. 

Would you be open to a short (1 hour) video consultation on your workflow / techniques?

Thanks,
Jake

**Comment by rl1987** [comment]
I don't really offer that. What do you want to ask?

**Comment by JakeWalker23** [comment]
It would be great to get a visual walkthrough of your end to end web scraping process, with tools and techniques. I just enquired about the consultancy as an indication I would be willing to pay you :) 

Thanks,
Jake
